components/atoms/forms/FileInput/FileInput.js

```diff
@@ -55,7 +55,7 @@ export default function FileInput({
   const handleChange = (event) => {
     if (event.target.files.length > 0) {
       updateFiles([...event.target.files], "add");
-      if (callback) callback(true);
+      if (onChange) onChange(true);
```

Comment:
pass `event` to `onChange` so that any parent component has access to the event as well